scripts/jenkins/ardana/ansible/files/rpm-added-files-whitelist.txt

```diff
@@ -0,0 +1,34 @@
+# alternatives system is expected to change stuff under /usr
+-> /etc/alternatives/.
+
+# Installed by osconfig/ansible/role/udev/tasks/configure.yml
+/usr/lib/systemd/system/ardana-udev\.service
+
+# FIXME: where does this come from?
```

Comment:
From http://git.suse.provo.cloud/cgit/ardana/ardana-ansible/tree/roles/installui/defaults/main.yml#n38 (used in one of the tasks)

Reply:
Thanks!

Reply:
This has since been moved to `/etc` but I think it should be moved to the rpm: https://bugzilla.suse.com/show_bug.cgi?id=1082708#c4